### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by Fubge**
Just fixes some small Typos

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Hi, I recently started my coding journey. This was my first pull requests and ive sadly overlocked the first part in the pull request guide that says "Do not edit files directly through GitHub".
I promise this mistake is not gonna happen any time in the future.
Sry :(

### CODE REVIEW

File: docs/how-to-enable-new-languages.md [hunk 1]
```diff
@@ -377,18 +377,18 @@ Then open a PR to the CDN repo to add both the YAML and Day.js files for review.
 
 ## Prep the News Repo for the New Language
 
-The [News repo](https://github.com/freeCodeCamp/news) pulls data from a Ghost instance, the files you added to the CDN, builds News, and deploys it.
+The [News repo](https://github.com/freeCodeCamp/news) pulls data from a Ghost Instance, the files you added to the CDN, builds News, and deploys it.
 
 > [!WARN]
-> Pull requests to the news repo _must_ come from the same repo. You should not work off of a fork for this step.
+> Pull requests to the News repo _must_ come from the same repo. You should not work off of a fork for this step.
```
Comment (Sboonny) [inline docs/how-to-enable-new-languages.md:383]: Welcome, @Fubge, and thank you for opening the PR.

I am not a native English speaker, so can you tell me why the other changes are typos?

I agree the "news" should be "News", but I don't see the need for the other changes.

**Comment by Sboonny** [comment]
Can you update the title to follow the PR naming convection https://contribute.freecodecamp.org/#/how-to-open-a-pull-request

File: docs/how-to-enable-new-languages.md (hunk 1 above)
Comment (Fubge) [reply docs/how-to-enable-new-languages.md:383]: Hello, Sboonny, and thank you for your reply.

Both spellings can be correct, but usually when using a specific term like "Ghost interface," both words should be capitalized ("Ghost Interface"). The same applies for "Crowdin Workflows."

I definitely made a mistake with the "project" change. Not capitalizing it is the only correct way in this context. I apologize for the error and appreciate your patience.

**Review by ShaunSHamilton — DISMISSED** [review]
Hello there,

Please revert all changes bar `news -> News`.

There is no such thing as a _"Ghost Instance"_. Only, an instance of _"Ghost"_.
There is no such thing as _"Crowdin Workflows"_. Only, workflows related to _"Crowdin"_.

In general, these proper nouns only make sense if there is a descriptor following. E.g.
> The "Curriculum Projects" **tab**.
> The "Crowdin Workflows" **settings**.

Again, these proper nouns do not exist. Hope this clarifies

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Fubge, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Pull request merged**